dsl/properties/scripts/ImportMicroservices.groovy

```diff
@@ -112,7 +112,11 @@ public class ImportMicroservices extends EFClient {
 
         // Service Fields
         efService.service.defaultCapacity = serviceConfig.deploy?.replicas
-        efService.service.minCapacity = serviceConfig.deploy?.updateConfig?.parallelism - serviceConfig.deploy?.replicas
```

Comment:
the fix described in the JIRA was for the 'Deploy Service' procedure where the logic should be:
parallelism = defaultCapacity - minCapacity
based on the definition of the parallelism attribute - number of updates that can happen in parallel which means this is the number of instances that will not be available. So we substract minCapacity from defaultCapacity to arrive at this value.
So, lets apply this logic to the 'Deploy Service' procedure at this location: https://github.com/electric-cloud/EC-Docker/blob/b858be4adcacd5823cd8334c973e85caa3c0fc94/dsl/properties/scripts/DockerClient.groovy#L579